templates/template-creation-web/src/engine/engine.ts

```diff
@@ -36,6 +37,9 @@ export class CreationEngine extends Application {
   /** Initialize the application */
   public async init(opts: Partial<ApplicationOptions>): Promise<void> {
     opts.resizeTo ??= window;
+    opts.resolution ??= getResolution();
+    opts.antialias ??= true;
```

Comment:
While i agree that turning anti aliasing on makes the template look better, my only reservation here is that this option impacts performance a lot, especially for mobile devices, and I wouldn't want new users falling into this trap

I'll have a think about this one and get back to you

Reply:
We can skip `antialias`, the resolution here is what makes difference the most

Reply:
Yeah let's skip`antialias` for now

Reply:
Removed